salt/modules/osquery.py

```diff
@@ -56,7 +56,7 @@ def _osquery(sql, format='json'):
     }
 
     cmd = 'osqueryi --json "{0}"'.format(sql)
-    res = __salt__['cmd.run_all'](cmd)
+    res = __salt__['cmd.run_all'](cmd, python_shell=True)
```

Comment:
I think this may be caused by the shlex.split in the cmd module. Just bypassing that for Windows might fix the issue; setting the posix option to false doesn't seem to account for quote marks and just executing the command as a single string should preserve them as-is.